applications/rag/metadata.yaml

```diff
@@ -17,27 +17,15 @@ spec:
   content: {}
   interfaces:
     variables:
-      - name: goog_cm_deployment_name
-        description: The name of the deployment.
-        varType: string
-        required: true
-      - name: add_auth
```

Comment:
Is this unused since showIf doesn't work?

Reply:
this is renmaed ot jupyter_add_auth & frontend_add_auth